zebra/zebra_router.c

```diff
@@ -164,8 +164,6 @@ static void zebra_router_free_table(struct zebra_router_table *zrt)
 {
 	void *table_info;
 
-	rib_close_table(zrt->table);
```

Comment:
is removing this a fix for a problem - some interaction with the zebra_vrf.c code for example? or is it a sort of "who's responsible" thing, consolidating responsibility in the vrf module?

Reply:
yes exactly.  This is a who's responsible problem